From patchwork Wed Jan  3 18:53:53 2018
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: hostapd: Allow HT40 on ch 165, 169.
X-Patchwork-Submitter: Ben Greear <greearb@candelatech.com>
X-Patchwork-Id: 855186
Message-Id: <1515005633-26836-1-git-send-email-greearb@candelatech.com>
To: hostap@lists.infradead.org
Cc: Ben Greear <greearb@candelatech.com>
Date: Wed,  3 Jan 2018 10:53:53 -0800
From: greearb@candelatech.com
List-Id: <hostap.lists.infradead.org>

From: Ben Greear <greearb@candelatech.com>

India supports CH 169 and 173 now.  Enable HT40 across
CH 165 and 169.  I think that CH 173 must remain HT20
only.

Signed-off-by: Ben Greear <greearb@candelatech.com>
---
 src/common/hw_features_common.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/common/hw_features_common.c b/src/common/hw_features_common.c
index 4486124..263958d 100644
--- a/src/common/hw_features_common.c
+++ b/src/common/hw_features_common.c
@@ -89,7 +89,7 @@ int allowed_ht40_channel_pair(struct hostapd_hw_modes *mode, int pri_chan,
 {
 	int ok, j, first;
 	int allowed[] = { 36, 44, 52, 60, 100, 108, 116, 124, 132, 140,
-			  149, 157, 184, 192 };
+			  149, 157, 165, 184, 192 };
 	size_t k;
 
 	if (pri_chan == sec_chan || !sec_chan)
